libs/components/data-manager/src/lib/modules/data-manager/data-manager-column-picker/data-manager-column-picker.component.html

```diff
@@ -25,9 +26,17 @@
     </sky-data-manager>
   </sky-modal-content>
   <sky-modal-footer>
+    <sky-alert
```

Comment:
An alert seems odd here. Did we check that with Adam? I would think that we would at least use a status indicator here instead of an alert based on our form design guidelines https://developer.blackbaud.com/skyux/design/guidelines/form-design#validation-and-error-handling

Reply:
resolved in the comments of a previous PR that is pulled into this: https://github.com/blackbaud/skyux/pull/348